src/index.tsx

```diff
@@ -20,6 +20,23 @@ import './app/i18n';
 document.body.classList.add(configClass, varsClass);
 settings.applyTheme();
 
+// Register Service Worker
+if ('serviceWorker' in navigator) {
+  navigator.serviceWorker.register(
+    import.meta.env.MODE === 'production' ? '/sw.js' : '/dev-sw.js?dev-sw'
```

Comment:
currently service worker file is loaded from the absolute root this may not work if cinny is hosted on the subpath, we should generate it like ``${trimTrailingSlash(import.meta.env.BASE_URL)}/sw.js``